app/Http/Resources/VenueResource.php

```diff
@@ -28,7 +28,8 @@ public function toArray($request)
                 'postcode' => $this->address->postcode,
                 'country' => $this->address->country
             ],
-            'atrributes' => $this->attributes->pluck('name')
```

Comment:
Create a new beverage resource and return a collection here so it would be like this:
```
'attributes' => AttributeResource::collection($this->attributes);
```